### PR TITLE
Add EMS basic configuration

### PR DESCRIPTION
**Pull request by colinbird**

### CODE REVIEW

File: openapi/components/schemas/GatewayAccountConfig/EMS.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+description: EMS config
```
Comment (colinbird) [inline openapi/components/schemas/GatewayAccountConfig/EMS.yaml:1]:
```suggestion
description: EMS e-Commerce (XML) API config
```

File: openapi/components/schemas/GatewayAccountConfig/EMS.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: EMS credentials object
+        properties:
+          storeId:
+            type: string
+            description: EMS merchant id
```
Comment (colinbird) [inline openapi/components/schemas/GatewayAccountConfig/EMS.yaml:14]:
```suggestion
            description: EMS store id
```

File: openapi/components/schemas/GatewayAccountConfig/EMS.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        properties:
+          storeId:
+            type: string
+            description: EMS store id
+          userId:
+            type: string
+            description: EMS account id
+          password:
+            type: string
+            description: EMS password
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/EMS.yaml:20]: format password?

Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/EMS.yaml:20]:
```suggestion
            description: EMS password
            format: password
```
?

File: openapi/components/schemas/Gateways/GatewayName.yaml [hunk 4]
```diff
@@ -39,6 +39,7 @@ enum:
   - EcorePay
   - Elavon
   - eMerchantPay
+  - EMS
```
Comment (ashkarpetin) [inline openapi/components/schemas/Gateways/GatewayName.yaml:42]: Add to AcquirerName?

Comment (colinbird) [reply openapi/components/schemas/Gateways/GatewayName.yaml:42]: It's already there

**Review by ashkarpetin — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/EMS.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            x-multiline: true
+          privateKeyPassword:
+            type: string
+            format: password
+            description: Private key password
+          clientCertificate:
+            type: string
+            description: Client Certificate
+            x-multiline: true
+          clientCrtificatePassword:
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/EMS.yaml:35]:
```suggestion
          clientCertificatePassword:
```

File: openapi/components/schemas/GatewayAccountConfig/EMS.yaml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+            description: Server Certificate
+            x-multiline: true
+        required:
+          - storeId
+          - userId
+          - password
+          - privateKey
+          - privateKeyPassword
+          - clientCertificate
+          - clientCertificatePassword
```
Comment (colinbird) [inline openapi/components/schemas/GatewayAccountConfig/EMS.yaml:50]:
```suggestion
          - clientCertificatePassword
          - serverCertificate
```

**Review by slavcodev — APPROVED** [review]

**Pull request merged**